### PR TITLE
Include all retro groups

### LINKED ISSUE #2862: Add all groups to agenda in retro
## Issue - Enhancement

[A user](https://app.hubspot.com/contacts/3888472/contact/3469901) writes:

> We were wondering why groups that don't get voted on don't show up on the side panel.  We had a retro with a lot of reflections today so some of the groups didn't get votes but the team still wanted to talk about them.  

Ideas to resolve this:

   A. Make it configurable at the start of the meeting
   B. Just add all the groups, and show they got zero votes
   C. Add a control to the bottom of the meeting to "add 0-vote groups"

Others?

My vote is probably just to start with B.

### PR DESCRIPTION
**Pull request by ackernaut**
Fixes #2862 

### Tests
- [ ] Create 5 reflections in a Retro meeting
- [ ] Vote for 3 reflections (casting 3 votes, 1 vote, and 1 vote)
- [ ] Advance to the Discuss phase
- [ ] See topics in sorted order 3, 1, 1, 0, 0 in the sidebar navigation
- [ ] Discuss phase view header (per topic) has correctly formatted vote count
- [ ] End the meeting
- [ ] The Quick Stats has a count for all topics
- [ ] There’s a section that shows all the topics
- [ ] From the timeline, look back at an previous Retro summary, it includes all topics

### CODE REVIEW

**Comment by ackernaut** [comment]
This is ready. I wonder if the smart thing to do would be to bundle any single-card groups with 0 votes into a group called “Single cards with 0 votes” but we could capture that idea as an issue, non-blocking for this PR.

File: packages/client/components/RetroDiscussPhase.tsx [hunk 1]
```diff
@@ -182,7 +182,7 @@ const RetroDiscussPhase = (props: Props) => {
                 <TopicHeading>{`“${title}”`}</TopicHeading>
                 <VoteMeta>
                   <VoteIcon>{meetingVoteIcon}</VoteIcon>
-                  {voteCount}
+                  {voteCount || '0'}
```
[the remark this answers is not in the document]
Comment (ackernaut) [reply packages/client/components/RetroDiscussPhase.tsx]: I didn’t look too hard at it, but I recall that when the value is 0 it doesn’t print anything. I did see that voteCount can be 0 in the data, and looked at the type declaration.

[the remark this answers is not in the document]
Comment (ackernaut) [reply packages/client/components/RetroDiscussPhase.tsx]: @mattkrick here’s the deal:
- [ ] run a retro including some cards/groups that don’t have votes
- [ ] advance to the discuss phase
- [ ] notice that the sidebar discuss section doesn’t have 0’s where there should be
- [ ] refresh, and the 0’s are there
So at some point a voteCount with value 0 is probably being interpreted as falsy?

[the remark this answers is not in the document]
Comment (mattkrick) [reply packages/client/components/RetroDiscussPhase.tsx]: ah, voteCount must not be coming from the server on update, just on the initial payload

[the remark this answers is not in the document]
Comment (ackernaut) [reply packages/client/components/RetroDiscussPhase.tsx]: When the Discuss phase first loads, `voteCount` is `undefined` for the groups that have 0 votes…going to look more closely at the graphql fragments.

[the remark this answers is not in the document]
Comment (ackernaut) [reply packages/client/components/RetroDiscussPhase.tsx]: Thanks for the pointer, added `voteCount` to query for 2 mutations

**Comment by ackernaut** [comment]
Okay @mattkrick ready again!

**Pull request merged**